fades/helpers.py

```diff
@@ -208,3 +210,37 @@ def check_pypi_updates(dependencies):
 
     dependencies["pypi"] = dependencies_up_to_date
     return dependencies
+
+
+def _pypi_head_package(pkg_name):
+    """Hit pypi with a http HEAD to check if pkg_name exists."""
+    url = BASE_PYPI_URL.format(name=pkg_name)
+    req = request.Request(url, method='HEAD')
+    try:
+        response = request.urlopen(req)
+    except HTTPError as http_error:
+        if http_error.code == HTTP_STATUS_NOT_FOUND:
+            return False
+        else:
+            raise http_error
+    else:
+        if response.status == HTTP_STATUS_OK:
+            logger.debug("%s exists in Pypi.", pkg_name)
+            return True
+
+
+def check_pypi_exists(dependencies):
+    """Check if the indicated dependencies actually exists in pypi."""
+    for dependency in dependencies.get('pypi', []):
+        logger.debug("Checking if %s exists in PyPi", dependency)
+        try:
+            exists = _pypi_head_package(dependency.project_name)
+        except Exception as error:
+            logger.error("Error checking %s in pypi: %r", dependency, error)
+            sys.exit(1)
+        else:
+            if not exists:
+                logger.error("%s doesn't exists in PyPi.", dependency)
```

Comment:
I'd log this in INFO, because is not our error or from the system. I thought about the user calling fades with --quiet and not seeing this, but note that we're already logging in error below, where we say "some dependency is not there, exiting"

Reply:
done